src/status_im/common/metrics_confirmation_modal/view.cljs

```diff
@@ -50,6 +52,15 @@
   (toggle-metrics false)
   (hide-bottom-sheet))
 
+(defn- on-privacy-policy-press
+  []
+  (rf/dispatch
+   [:show-bottom-sheet
+    {:content (fn []
+                [quo.theme/provider :dark
+                 [privacy/privacy-statement]])
```

Comment:
The theme provider is set as `:dark` here because the `privacy-statement` component renders its text depending on the theme in the current context. 

For some reason, the react context here says that the theme is `:light` so the text for the privacy policy is rendered in black.

 This change makes the privacy policy text to be rendered in white making sure we don't impact another area.